apps/design/frontend/src/contests_screen.tsx

```diff
@@ -740,15 +740,49 @@ function ContestForm({
       )}
 
       {contest.type === 'yesno' && (
-        <div>
-          <FieldName>Description</FieldName>
-          <RichTextEditor
-            initialHtmlContent={contest.description}
-            onChange={(htmlContent) =>
-              setContest({ ...contest, description: htmlContent })
-            }
-          />
-        </div>
+        <React.Fragment>
+          <div>
+            <FieldName>Description</FieldName>
+            <RichTextEditor
+              initialHtmlContent={contest.description}
+              onChange={(htmlContent) =>
+                setContest({ ...contest, description: htmlContent })
+              }
+            />
+          </div>
+
+          <div>
+            <FieldName>&quot;Yes&quot; Option Label</FieldName>
```

Comment:
What do you think of "First Option Label" and "Second Option Label" instead?

Knowing that the one town to soon use this will enter in "A" / "B" - "Yes Option Label" may be a confusing title for that